pdctl/command/config_command.go

```diff
@@ -62,19 +60,12 @@ func NewSetConfigCommand() *cobra.Command {
 }
 
 func showConfigCommandFunc(cmd *cobra.Command, args []string) {
-	url := getAddressFromCmd(cmd, schedulePrefix)
-	r, err := http.Get(url)
+	r, err := doRequest(cmd, schedulePrefix, "GET")
```

Comment:
Could you use `http.MethodGet`, which is already define in `net/http`?

https://golang.org/pkg/net/http/#pkg-constants